### PR TITLE
Docs update for unauthorized studies portal prop

### PR DESCRIPTION
**Pull request by dianab0**
Updated portal property reference docs for the configuration of a global message for unauthorized studies.

This docs update is related to PR https://github.com/cBioPortal/cbioportal-frontend/pull/4138

### CODE REVIEW

**Review by pvannierop — CHANGES_REQUESTED** [review]

File: docs/portal.properties-Reference.md [hunk 1]
```diff
@@ -348,6 +351,11 @@ The default view in OncoPrint ("patient" or "sample") can be set with the follow
 oncoprint.defaultview=sample
 ```
 
+Configuration of tracks that will be visible by default in the oncoprint. It points to a JSON file on the classpath.
+```
+oncoprint.clinical_tracks.show_by_default=classpath:/oncoprint-default-tracks.json
+```
+
```
[the remark this answers is not in the document]
Comment (dianab0) [reply docs/portal.properties-Reference.md]: Removed

File: docs/portal.properties-Reference.md [hunk 2]
```diff
@@ -176,7 +176,10 @@ Study View.
...
+If show_unauthorized_studies has been enabled, then a global message for the studies that are unauthorized can be configured by the property below. The global message will inform the user of insufficient permissions and will appear inside a tooltip when hover over the lock icon of the study.
+This message can also contain study specific information using the study tags. The information in the study tags can be accessed using Json Path placeholders, as for example {$.Owner.email}. For the studies that don't have this information available in the study tags, the default global message "The study is unauthorized. You need to request access." will be displayed. In addition to the study tags information, the study id can be included as well in the message using {$.studyiD} placeholder.
+For example, the property can be defined as below:
+skin.home_page.unauthorized_studies_global_message=The study {$.studyId} is unauthorized. You need to request access to {$.Owner.email}.
```
[the remark this answers is not in the document]
Comment (dianab0) [reply docs/portal.properties-Reference.md]: Done

**Comment by dianab0** [comment]
Added changes from review under the same commit

**Review by dippindots — APPROVED** [review]
@dianab0 Looks good to me, Are the related changes already merged? Could you mention that pr in the description if it's already merged?

**Comment by dianab0** [comment]
@dippindots , thank you, no, the PR containing related changes wasn't merged yet https://github.com/cBioPortal/cbioportal-frontend/pull/4138

**Review by pvannierop — APPROVED** [review]
Looks good.

**Comment by dianab0** [comment]
@dippindots , the PR containing related changes (https://github.com/cBioPortal/cbioportal-frontend/pull/4171) has been merged, so this one is ready to be merged as well. Thank you.

**Pull request merged**